src/handlers/payout/action.ts

```diff
@@ -100,14 +99,13 @@ export const handleIssueClosed = async () => {
           logger.info(`Skipping to generate permit2 url, reason: autoPayMode for this issue: false`);
           return `Permit generation skipped since autoPayMode for **THIS ISSUE** is disabled`;
         }
-        return;
       }
     }
   }
 
-  if (!autoPayMode) {
-    logger.info(`Skipping to generate permit2 url, reason: { autoPayMode: ${autoPayMode}}`);
-    return `Permit generation skipped since autoPayMode is disabled`;
+  if (paymentPermitMaxPrice == 0) {
```

Comment:
Please make sure to handle if this `paymentPermitMaxPrice` is `undefined` in the config? Unless it will automatically default to `0` if it is not defined.